addons/web_editor/static/src/js/editor/odoo-editor/test/spec/copyPaste.test.js

```diff
@@ -768,6 +761,150 @@ describe('Paste', () => {
             });
         });
     });
+    describe('Simple html elements containing <br>', () => {
+        describe('breaking <br> elements', () => {
+            it('should split h1 with <br> into seperate h1 elements', async () => {
+                await testEditor(BasicEditor, {
+                    contentBefore: '<p>[]<br></p>',
+                    stepFunction: async editor => {
+                        await pasteHtml(editor, '<h1>abc<br>def<br>ghi<br>jkl</h1>');
+                    },
+                    contentAfter: '<p>abc</p><h1>def</h1><h1>ghi</h1><p>jkl[]<br></p>',
```

Comment:
This is an incorrect result which gets fixed with your next commit. It would be better to swap your two commits to you can directly write the correct test case.